library/testing/TestSDKOptions.cxx

```diff
@@ -142,7 +142,7 @@ int TestSDKOptions(int argc, char* argv[])
 
   std::stringstream ssDir;
   ssDir << f3d::direction_t(0, 0, -1.0);
-  test("direction to string", ssDir.str() == "0,0,-1");
+  test("direction to string", ssDir.str(), "-Z");
```

Comment:
maybe add a test above checking the code path that format to the vector format ?

Reply:
all good, you did it in OptionIO (which is the right place)